ui/src/types/templates.ts

```diff
@@ -186,3 +185,5 @@ export interface VariableTemplate extends TemplateBase {
 }
 
 export type Template = TaskTemplate | DashboardTemplate | VariableTemplate
+
+export {TemplateType} from '@influxdata/influx'
```

Comment:
nit: I'd expect this at the top of this file.

Reply:
We have traditionally put our exports at the bottom of files.